code/modules/power/fission/core.dm

```diff
@@ -0,0 +1,182 @@
+#define ACTIVE_THRESHOLD             200
```

Comment:
is all the fission stuff intended to be in this PR? I see some related bits where controller is, but that's a lot of fissionstuff so unsure

Reply:
Nope, my mistake. Accidentally staged the files when I was rebasing. Should be fixed now.